net/ostiary/files/ostiaryd.init

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh /etc/rc.common
+
+START=99
+STOP=85
+
+USE_PROCD=1
+
+start_service()
+{
+
+	procd_open_instance
+	procd_set_param respawn
+	procd_set_param command /usr/bin/ostiaryd -c /etc/ostiary.cfg 
+	procd_close_instance
+}
+
+stop_service()
+{
+	killall ostiaryd
+}
+
+reload_service()
+{
+restart
+}
+
+service_triggers()
```

Comment:
Reload/service trigger like 'procd_add_reload_trigger' depends on uci config files, therefore you can remove this as well.

Reply:
removed